README.md

```diff
@@ -1,2 +1,254 @@
+      <xs:attribute name="interpreter" use="required" type="xs:IDREF"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="datasources">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element maxOccurs="unbounded" minOccurs="1" ref="datasource"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="datasource">
+    <xs:complexType>
+      <xs:attribute name="class" use="required"/>
+      <xs:attribute name="id" use="required" type="xs:ID"/>
+      <xs:attribute name="monitor" use="required"/>
+      <xs:attribute name="options" use="required"/>
+      <xs:attribute name="type" use="required" type="xs:NCName"/>
+      <xs:attribute name="uri" use="required" type="xs:anyURI"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="interpreters">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element maxOccurs="unbounded" minOccurs="1" ref="interpreter"/>
+      </xs:sequence>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="interpreter">
+    <xs:complexType>
+      <xs:sequence>
+        <xs:element maxOccurs="unbounded" minOccurs="1" ref="configuration"/>
+      </xs:sequence>
+      <xs:attribute name="id" use="required" type="xs:ID"/>
+    </xs:complexType>
+  </xs:element>
+  <xs:element name="configuration">
+    <xs:complexType>
+      <xs:attribute name="analyser" use="required"/>
+      <xs:attribute name="parser" use="required"/>
+      <xs:attribute name="type" use="required"/>
+    </xs:complexType>
+  </xs:element>
+</xs:schema>
+```
+
+**monitoring-config.xml**
+```
+<?xml version="1.0" encoding="utf-8"?>
+<config xmlns="http://www.atb-bremen.de"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.atb-bremen.de monitoring-config.xsd">
+
+    <indexes>
+        <index id="index-git" location="target/indexes/git"/>
+    </indexes>
+
+    <datasources>
+        <datasource id="datasource-git" type="messagebroker"
+                    monitor="de.atb.context.monitoring.monitors.GitMonitor"
+                    uri=""
+                    options="server=localhost&amp;port=5672&amp;exchange=smartclide-monitoring&amp;topic=monitoring.git.*&amp;dle-topic=dle.git.commits"
+                    class="de.atb.context.monitoring.config.models.datasources.MessageBrokerDataSource"/>
+    </datasources>
+
+    <interpreters>
+        <interpreter id="interpreter-git">
+            <configuration type="*"
+                           parser="de.atb.context.monitoring.parser.GitParser"
+                           analyser="de.atb.context.monitoring.analyser.GitAnalyser"/>
+        </interpreter>
+    </interpreters>
+
+    <monitors>
+        <monitor id="monitor-git" datasource="datasource-git" interpreter="interpreter-git" index="index-git"/>
+    </monitors>
+</config>
+```
+
+The "monitoring-config.xml" configuration file can also be changed by using the Admin UI.
```

Comment:
Some more information how to access the Admin UI would be good.

Reply:
I think this should be remove for now, because the UI needs major changes